app/brave_generated_resources.grd

```diff
@@ -932,11 +932,19 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Maybe later
       </message>
 
-      <if expr="is_win">
-        <message name="IDS_FIRSTRUN_DLG_PIN_SHORTCUT_TEXT" desc="Text for pin to taskbar checkbox">
-          Pin to taskbar
-        </message>
+      <if expr="enable_pin_shortcut">
+        <if expr="is_win">
+          <message name="IDS_FIRSTRUN_DLG_PIN_SHORTCUT_TEXT" desc="Text for pin to taskbar checkbox">
+            Pin to taskbar
+          </message>
+        </if>
+        <if expr="is_macosx">
+          <message name="IDS_FIRSTRUN_DLG_PIN_SHORTCUT_TEXT" desc="Text for pin to taskbar checkbox">
+            Keep in dock
```

Comment:
"dock" or "Dock" or "the Dock"? (also applies to other strings)

I believe it should be "the Dock":
https://support.apple.com/guide/mac-help/open-apps-from-the-dock-mh35859/mac
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/mac/dock.h?q=the%20dock&ss=chromium

Reply:
Ok, let's wait @rmcfadden3 's review about the labels in settings and dialog.

Reply:
@simonhong — per how Apple does it, it appears it should be "Dock" capitalized, and in the instance of settings copy, the article "the" is not needed (see screenshot below).

So I'd think the copy should be:
- Keep in Dock
- Brave is already in Dock
- 
![Screen Shot 2022-10-27 at 2 33 33 PM](https://user-images.githubusercontent.com/78937892/198402755-8e38e655-93b1-4c8a-a345-f6d9e80c5a25.png)

Reply:
@rmcfadden3 Updated. Thanks! 👍🏼